src/methods.jl

```diff
@@ -1,5 +1,128 @@
 export FDMReport, fdm, backward_fdm, forward_fdm, central_fdm
 
+"""
+    FDM.DEFAULT_CONDITION
+
+The default [condition number](https://en.wikipedia.org/wiki/Condition_number) used when
+computing bounds. It provides amplification of the ∞-norm when passed to the function's
+derivatives.
+"""
+const DEFAULT_CONDITION = 100
+
+"""
+    FDM.TINY
+
+A tiny number added to some quantities to ensure that division by 0 does not occur.
+"""
+const TINY = 1e-20
```

Comment:
Maybe we want to replace this constant by `eps()`? Or have I misunderstood what this is going to be used for?

Reply:
Yeah, we could do that. I was just following what's done in the Python implementation.

Reply:
The tiny number here should be absolutely small, so I'm not sure `eps()` makes sense. What we had before, `eps(0.0)`, would be ideal, if it not were at the edge of the floating point range. `1e-20` is a compromise to have something absolutely small to what we usually work with, whilst not it being too small to cause issues with the numerics.

Reply:
Perhaps the above comments doesn't really make sense. `TINY` is supposed to be small compared to any number that we typically encounter, but `eps()` is only small to numbers of the order of magnitude of `1`. If we consider `1e-6` on the lower end of sizes of numbers we typically encounter, then `eps(1e-6) = 2e-22` should be sufficient, which is where I originally got the `1e-20` from. Does that make more sense? 😬